### DISCUSSION TITLE
Assistance for the Community Scripts

### DISCUSSION THREAD

**Discussion by MrThorx** [community-scripts/ProxmoxVE#28] [General]
Hello,

sorry in advance for the possibly stupid question.

I have previously used the proxmox helper scripts directly via the tteck website. I was surprised to read the post that this is now becoming a community project due to tteck's health.

on tteck's website it was easy for me to click the copy button and follow the steps in proxmox.

what about the community project now, I'm honestly a complete newbie. Can I continue to use the scripts via copy from the "normal" website?

what changes with the community project, how can I then use the scripts?

thanks in advance for the help.

regards

thorx

**Comment by MickLesk** [comment]
Hey!

You can take a look here:
https://github.com/community-scripts/ProxmoxVE/discussions/13#discussioncomment-11131675
(I work on an workaround next days)

If you Open the helper-scripts Site, it will be redirected to:
https://community-scripts.github.io/Proxmox/

There are the new fork.

**Comment by MrThorx** [comment]
thanks a lot :-)

**Reply by tteck** [reply]
But, the OG site (https://tteck.github.io/Proxmox/) is secure and fully functional. 😉
